elastic/kibana
# Rule detail table with bulk actions

## Summary

Closes #136520

### QA

- Create Rule and Alert in o11y
- Go to rule detail page in o11y (o11y - alerts - manage rules and select a rule)
- Click on Alerts tab
- You should see the checkbox column and when at least one item is selected the toolbar with all the bulk actions as follows

<img width="1723" alt="Screenshot 2022-07-22 at 11 19 10" src="https://user-images.githubusercontent.com/17549662/180407758-bf80c943-d432-44d1-9f10-8061e977c886.png">

- You should see the in row actions

<img width="1723" alt="Screenshot 2022-07-22 at 11 17 46" src="https://user-images.githubusercontent.com/17549662/180407505-bdd08b34-d05b-48a6-a84e-0532ff6969cc.png">

### Issues

Please feel free to let me know how you consider we should proceed with this next issues:

1.  Consider adding a functional test for this integration, we should be covered from our side with all the unit testing we have done but it might be useful to check that the whole integration is working. 
2. We have seen that using the current row actions include a `View rule details` option which we think should be removed for the rule detail page as the user will be already there so it won't do anything

<img width="1125" alt="Screenshot 2022-07-21 at 16 22 18" src="https://user-images.githubusercontent.com/17549662/180238459-6b9f387c-dc2e-4c99-8d6c-9e692ef9bcd7.png">
